configs/scripts/program/dump.sh

```
@@ -29,56 +31,93 @@ if [ ! -d ${OUTPUT} ]; then
 
-# dump external programs binaries if needed
-for i in ${!EXTERNAL_ID[@]}; do
-    if [ ! -f "${OUTPUT}/${EXTERNAL_SO[$i]}" ]; then
-        copy_from_chain "${EXTERNAL_SO[$i]}"
+# only prints this if we have mainnet external programs
+if [ ${#EXTERNAL_ID_MAINNET[@]} -gt 0 ]; then
+    echo "Dumping external accounts from mainnet to '${OUTPUT}':"
+fi
+
+# dump mainnet external programs binaries if needed
+for i in ${!EXTERNAL_ID_MAINNET[@]}; do
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)